va/va.h

```diff
@@ -794,6 +811,28 @@ typedef enum
      *  attribute value is \c VAConfigAttribValContextPriority
      */
     VAConfigAttribContextPriority       = 41,
+    /**
+     * \brief VAConfigAttribDisplayLinkHdcpWired
+     *
+     * This attribute specifies wired HDCP settings for protected display link.
+     * 
+     **/
+    VAConfigAttribDisplayLinkHdcpWired  = 42,
+    /**
+     * \brief VAConfigAttribDisplayLinkHdcpWireless
+     *
+     * This attribute specifies wireless HDCP settings for protected display link.
```

Comment:
same.

Reply:
Remove this config attribute since it will not be used.